src/main/dynamo/Table.ts

```diff
@@ -12,14 +12,18 @@ export class Table {
   constructor(config: {
     name: string
     partitionKeyName: string
-    sortKeyName: string
+    sortKeyName: string,
+    encryptionBlacklist?: string[]
```

Comment:
This could be `encryptionBlacklist: string[] = []` Then you won't have to do `|| []` below

Reply:
`encryptionBlacklist: string[] = [] ` will generate syntax error here because we're defining a `type` for config variable.

Reply:
Ah sorry missed the `config: {...}` part at the top